content/hi/training/_index.html

```diff
@@ -0,0 +1,154 @@
+---
+title: प्रशिक्षण
+bigheader: कुबेरनेट्स प्रशिक्षण और प्रमाणन
+abstract: प्रशिक्षण कार्यक्रम, प्रमाणपत्र, और भागीदार।
+layout: basic
+cid: training
+class: training
+---
+
+<section class="call-to-action">
+  <div class="main-section">
+    <div class="call-to-action" id="cta-certification">
+      <div class="cta-text">
+        <h2>अपना क्लाउड नेटिव करियर बनाएं</h2>
+        <p>कुबेरनेट्स क्लाउड नेटिव आंदोलन के मूल में है। लिनक्स फाउंडेशन और हमारे प्रशिक्षण भागीदारों से प्रशिक्षण और प्रमाणन आपको अपने करियर में निवेश करने, कुबेरनेट्स सीखने और अपनी क्लाउड नेटिव परियोजनाओं को सफल बनाने की सुविधा देता है।</p>
+      </div>
+      <div class="cta-img-container">
+        <div class="logo-certification cta-image" id="logo-kcnf">
+          <img src="/images/training/kubernetes-kcnf-white.svg" />
+        </div>
+        <div class="logo-certification cta-image" id="logo-kcsa">
+          <img src="/images/training/kubernetes-cksa-white.svg" />
+        </div>
+        <div class="logo-certification cta-image" id="logo-cka">
+          <img src="/images/training/kubernetes-cka-white.svg"/>
+        </div>
+        <div class="logo-certification cta-image" id="logo-ckad">
+          <img src="/images/training/kubernetes-ckad-white.svg"/>
+        </div>
+        <div class="logo-certification cta-image" id="logo-cks">
+          <img src="/images/training/kubernetes-cks-white.svg"/>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section>
+  <div class="main-section padded">
+    <center>
+      <h2>edX पर निःशुल्क पाठ्यक्रम लें</h2>
+    </center>
+  <div class="col-container">
+    <div class="col-nav">
+      <center>
+        <h5>
+          <b>कुबेरनेट्स का परिचय <br> &nbsp;</b>
+        </h5>
+        <p>क्या आप कुबेरनेट्स सीखना चाहते हैं?? कंटेनरीकृत अनुप्रयोगों के प्रबंधन के लिए इस शक्तिशाली प्रणाली पर गहन प्राइमर प्राप्त करें।</p>
```

Comment:
This line is not clear to understand, due to use of some pure Hindi words at the end of sentence. I suggest replacing those words, with words that we spoke in our daily life.